src/app/example-components/card/card.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { timer } from 'rxjs';
+import { finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent {
+  isInEditMode = false;
+  firstName = 'Jane';
+  lastName = 'Doe';
+  
+  onCancel(): void {
+    // reset values
+    this.firstName = 'Jane';
+    this.lastName = 'Doe';
+    this.isInEditMode = false;
+  }
+
+  onSave(): void {
+    // simulate a HTTP call to the backend
```

Comment:
Da ist mir aufgefallen, dass ich während ich warte trotzdem noch alles machen kann, da sollte man die Action-Buttons fast deaktivieren bzw. eine Loading indication einbauen. Aber das kommt halt darauf an, ob in der aufrufenden Komponente eine Verzögerung beim Speichern (z.B. durch Backend Call) des Edit Modes gibt oder nicht.